### PR TITLE
ThreatMiner added reliability

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: demisto/etc#29347

## Description
Added reliability

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ThreatMiner/Integrations/ThreatMiner/ThreatMiner.py [hunk 1]
```diff
@@ -665,29 +682,44 @@ def delete_proxy_if_asked():
...
         else:
             demisto.results('test failed')
 
-    if demisto.params().get('limit_results').lower() == 'all':
-        max_array_size = -1
+    if params.get('limit_results').lower() == 'all':
+        args['max_array_size'] = -1
     else:
-        max_array_size = int(demisto.params().get('limit_results', MAX_RETURNED_ARRAY_SIZE))
+        args['max_array_size'] = int(demisto.params().get('limit_results', 30))
```
Comment (JasBeilin) [inline Packs/ThreatMiner/Integrations/ThreatMiner/ThreatMiner.py:713]: replace demisto.params to params

**Review by JasBeilin — COMMENTED** [review]
Lets change the README as well, and lets take another look at the tests.

**Review by bakatzir — CHANGES_REQUESTED** [review]
Good work adding the missing outputs in the yml.
regarding the args and params handling - I think that it is not a good practice to unify the integration params(integrationReliability, insecure) into the args dict that hold the arguments.
please:
keep one for params, 1 for args, move and both to each func
or
move just args and create a class with the params and properties and http_request as a func of the class(which is the preferred sol IMO).

File: Packs/ThreatMiner/Integrations/ThreatMiner/ThreatMiner.py [hunk 2]
```diff
@@ -664,34 +681,55 @@ def delete_proxy_if_asked():
...
+
+        if demisto_command == 'file':
+            file_command(**params)
+
+    except Exception as e:
+        demisto.results({
+            'Type': entryTypes['error'],
+            'ContentsFormat': formats['text'],
+            'Contents': 'error has occured: %s' % (e.message, ),
+        })
```
Comment (bakatzir) [inline Packs/ThreatMiner/Integrations/ThreatMiner/ThreatMiner.py:731]:
```suggestion
        return_error('An error has occurred: %s' % (e.message))
```

File: Packs/ThreatMiner/Integrations/ThreatMiner/Threatminer_test.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+DBOT_SCORES = {
+    'Reliability': 'C - Fairly reliable',
+    'Vendor': 'ThreatMiner',
+    'Indicator': 'CA978112CA1BBDCAFAC231B39A23DC4DA786EFF8147C4E72B9807785AFEE48BB',
+    'Score': 0,
+    'Type': 'File'
+}
+
+
+def test_reliability_in_dbot(mocker):
```
Comment (bakatzir) [inline Packs/ThreatMiner/Integrations/ThreatMiner/Threatminer_test.py:14]: please conform to the when/given/then format

Comment (yaakovpraisler) [reply Packs/ThreatMiner/Integrations/ThreatMiner/Threatminer_test.py:14]: done.

**Review by bakatzir — APPROVED** [review]

**Pull request merged**